test/opentelemetry/solarwinds_processor_test.rb

```diff
@@ -1,85 +0,0 @@
-# frozen_string_literal: true
```

Comment:
Seems the tests in this file should be moved to the otlp processor testing, are you planning to add them later?

Reply:
Updated

Reply:
Hmm, not seeing them... are they supposed to be in the otlp processor test file?

Reply:
I put them in the file `solarwinds_otlp_processor_test.rb`.